rector.php

```diff
@@ -112,8 +112,8 @@
     $rectorConfig->rule(SimplifyEmptyArrayCheckRector::class);
     $rectorConfig->rule(NormalizeNamespaceByPSR4ComposerAutoloadRector::class);
     $rectorConfig
-        ->ruleWithConfiguration(TypedPropertyRector::class, [
+        ->ruleWithConfiguration(TypedPropertyFromAssignsRector::class, [
```

Comment:
I updated deprecated TypedPropertyRector to TypedPropertyFromAssignsRector

Reply:
Thank you!